core/test/audits/mainthread-work-breakdown-test.js

```diff
@@ -91,47 +119,57 @@ Object {
     expect(Math.round(output.numericValue)).toMatchInlineSnapshot(`979`);
     assert.equal(output.details.items.length, 7);
     assert.equal(output.score, 1);
-  });
-
-  it('should compute the correct values for the load trace (legacy)', async () => {
-    assert(loadTraceOld.find(e => e.name === 'TracingStartedInPage'));
-    const artifacts = {traces: {defaultPass: {traceEvents: loadTraceOld}}};
-
-    const output = await PageExecutionTimings.audit(artifacts, {options, computedCache: new Map()});
-    assert.deepStrictEqual(keyOutput(output), loadTraceExpectations);
-    assert.equal(Math.round(output.numericValue), 933);
-    assert.equal(output.details.items.length, 6);
-    assert.equal(output.score, 1);
+    expect(output.metricSavings.TBT).toBeCloseTo(353.5, 0.1);
   });
 
   it('should compute the correct values for the load trace', async () => {
     assert(loadTrace.traceEvents.find(e => e.name === 'TracingStartedInBrowser'));
-    const artifacts = {traces: {defaultPass: loadTrace}};
+    const artifacts = {
+      traces: {defaultPass: loadTrace},
+      devtoolsLogs: {defaultPass: loadDevtoolsLog},
+      URL: getURLArtifactFromDevtoolsLog(loadDevtoolsLog),
+      GatherContext: {gatherMode: 'navigation'},
+    };
 
-    const output = await PageExecutionTimings.audit(artifacts, {options, computedCache: new Map()});
+    context.settings.throttlingMethod = 'simulate';
+
+    const output = await PageExecutionTimings.audit(artifacts, context);
     expect(keyOutput(output)).toMatchInlineSnapshot(`
 Object {
-  "other": 59,
-  "paintCompositeRender": 48,
-  "parseHTML": 3,
-  "scriptEvaluation": 11,
-  "scriptParseCompile": 1,
-  "styleLayout": 103,
+  "other": 267,
+  "paintCompositeRender": 130,
+  "parseHTML": 2,
+  "scriptEvaluation": 12,
+  "scriptParseCompile": 2,
+  "styleLayout": 363,
 }
 `);
-    expect(Math.round(output.numericValue)).toMatchInlineSnapshot(`224`);
+    expect(Math.round(output.numericValue)).toMatchInlineSnapshot(`775`);
```

Comment:
Before this patch, `throttlingMethod` had a value of `undefined` so we didn't apply the 4x CPU multiplier to the timings.

In this patch we need to specify the same `throttlingMethod` that was used to collect the trace. In this case it was `simulate`